tests/test_usgs_stac.py

```diff
@@ -0,0 +1,40 @@
+import os
+import unittest
+from stactools.landsat.usgs_stac import convert_usgs_stac
+
+
+class UsgsStacTest(unittest.TestCase):
+
+    def test_tm(self):
+        mtl_xml_href = "tests/data-files/usgs-stac/tm/LT04_L2SP_002026_19830110_20200918_02_T1_MTL.xml"  # noqa
```

Comment:
To guard against relative filepath issues if tests are run from the non-root directory, use https://github.com/stac-utils/stactools/blob/main/src/stactools/testing/test_data.py#L22.

Reply:
Ah. I see and understand. Will do.